### DISCUSSION TITLE
有foobar2000的插件吗？

### DISCUSSION THREAD

**Discussion by NathanShi** [amll-dev/applemusic-like-lyrics#272] [General]
如题，我似乎没找到对应的foobar2000插件，Mac版的foobar2000默认无滚动歌词，若能得到该项目的支持就好了。多谢作者！
如果已提供/正在开发对应插件，请忽略此条discussion，我对于我给您带来的不便与打扰深表歉意。
注：在Mac下，AMLL.Player貌似不能播放32位flac&导入多个歌曲比较麻烦（歌曲目录格式为：艺人文件夹-->专辑文件夹-->歌曲文件，AMLL.Player目前似乎不能直接导入文件夹），所以我又用回了foobar2000。

**Comment by Linho1219** [comment]
暂无相关开发计划。感谢理解。
